packages/back/src/routes.js

```diff
@@ -34,7 +35,10 @@ module.exports = (app) => {
   app.delete('/users/:id', [authenticate, isAdmin, mongoQueries], deleteUser);
   app.get('/users', [authenticate, isAdmin, mongoQueries], getUsers);
   // Animals
+  // TODO: add authentication to this apis
```

Comment:
this is already done, can be done by adding the `authenticate` middleware in the routes file